examples/read_write.rs

```diff
@@ -0,0 +1,148 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+use std::process::Command;
+
+use ceresdb_client_rs::{
+    client::{Client, RpcContext},
+    model::{request::QueryRequest, value::Value, write::WriteRequestBuilder},
+    Builder, DbClient,
+};
+use chrono::Local;
+
+fn create_table() {
```

Comment:
Actually, we can use query method provided the sdk to do DDL and how about creating/dropping table by the SDK?